startupscript/butane/aws/metadata-utils.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# metadata-utils.sh defines helper functions for aws ec2 tags. This script is intended to be sourced from other scripts
+# to retrieve or modify instance tags. It is run on the VM host. AWS CLI is not installed on Flatcar VM by default so
+# we are running the AWS CLI in a container.
+
+# Defines a function to retrieve an instance tag set on the VM.
+function get_metadata_value() {
+  if [[ -z "$1" ]]; then
+    echo "usage: get_metadata_value <tag>"
+    exit 1
+  fi
+  local tag_key=vwbapp:"$1"
+
+  INSTANCE_ID="$(wget -q -O - http://169.254.169.254/latest/meta-data/instance-id)"
+  docker run --rm public.ecr.aws/aws-cli/aws-cli \
```

Comment:
Should we run these `aws` commands with `--network host`, at least until @nfoen propagates the change that sets the hop count to 2 for IMDS?